pkg/gatherers/clusterconfig/node_logs.go

```diff
@@ -0,0 +1,149 @@
+package clusterconfig
+
+import (
+	"bufio"
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/openshift/insights-operator/pkg/gatherers/common"
```

Comment:
I thought we had a convention where we have 3 groups of imports: from the standard library, external packages, operator's code

Reply:
What do you mean by that?

Reply:
I mean that I would group imports like this:
```
    "bufio"
	"bytes"
	"compress/gzip"
	"context"
	"fmt"
	"io"
	"strconv"

	corev1 "k8s.io/api/core/v1"
	"k8s.io/klog/v2"
	"k8s.io/client-go/rest"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	"k8s.io/client-go/kubernetes"
	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"

	"github.com/openshift/insights-operator/pkg/gatherers/common"
	"github.com/openshift/insights-operator/pkg/recorder"
	"github.com/openshift/insights-operator/pkg/utils/marshal"
	"github.com/openshift/insights-operator/pkg/record"
```
but maybe we don't have any convention for it

Reply:
I think we don't...but it's true that it's grouped differently than here. I mean into 3 groups (as Serhii mentioned) where the last two somehow intersect. Isn't this part of `go fmt` or our linting??

Reply:
Gotcha! I can check if it's possible to add this validation to the lining. I think that the pre-commit should format the code before pushing it. I need to check why it is not happening.

Reply:
OK. So, a quick investigation point that we have two different styles of coding format to import: [check here](https://github.com/golang/go/issues/37719) and [here](https://goinbigdata.com/goimports-vs-gofmt/). If we want to use the one by goimports I'll check how to do linting it and add it to the pre-commit script later. For now, the better approach is to configure the IDE to do it.